docs/root/intro/arch_overview/advanced/attributes.rst

```diff
@@ -0,0 +1,153 @@
+.. _arch_overview_attributes:
+
+Attributes
+==========
+
+Attributes refer to contextual properties provided by Envoy during request and
+connection processing. They are named by a dot-separated path (e.g.
+`request.path`), have a fixed type (e.g. `string` or `int64`), and may be
+absent or present depending on the context. Attributes are exposed to CEL
+runtime in :ref:`RBAC filter <arch_overview_rbac>`, as well as Wasm extensions
+via `get_property` ABI method.
+
+Request attributes
+------------------
+
+The following request attributes are generally available upon initial request
```

Comment:
It might be worth describing the value type. It's not a string, as it might be map. In https://github.com/envoyproxy/envoy/pull/13893 its a Struct, etc.

Reply:
Are you referring to `request` itself? I want to leave it unspecified for now, since it's not typical to operate on the entire request right now, and we don't want to force protobuf representation for it. Internally, this is a `CelMap` which is sort of like typed JSON. The main difference between CelMap and protobuf is lack of default values, so missing fields would be an error instead of a default value.

Reply:
No, I'm referring to header maps. Having some formal type mapping as you mention in your other comment would help clear this up.

Reply:
Explained that they are maps and values are comma-concatenated.